doc/rho.1

```diff
@@ -0,0 +1,408 @@
+.TH rho 1 "August 10, 2017" "version 0.1" "rho User Commands"
+.SH NAME
+rho \- Easily discover and manage systems on your network.
+
+.SH SYNOPSIS
+rho
+.I command subcommand [options]
+
+.SH DESCRIPTION
+rho is a network discovery tool to identify the number of systems running on a network, their operating system, hardware, and release numbers for almost any Linux or Unix flavor. Being able to identify the systems running on the network is a vital component to managing licenses and renewals. Ultimately, discovery is part of the larger sysadmin task of managing inventories.
+.PP
+rho uses three configuration entries to manage the discovery process.
+.I Profiles
+define the network or subnet that is being monitored.
+.I Credentials
+contains the usernames, passwords or, alternatively, the SSH keys of the user as which discovery runs. There can be multiple network profiles and authentication credentials, used in any combination.
+.I Reports
+define the fields and output filename that will be run with a scan.
+.PP
+rho is an
+.I agentless
+discovery tool, so there is no need to install anything on multiple systems. OS discovery for the entire network is centralized to a single machine.
+.PP
+This man page covers the commands, subcommands and options for rho with basic usage information. For more detailed information and examples, including best practices, see the rho README.
+
+.SH USAGE
+rho performs four major tasks:
+.PP
+1. Creating authentication profiles. This has the basic command:
+.PP
+.B rho auth add ...
+.PP
+2. Creating network profiles. This has the basic command:
+.PP
+.B rho profiles add --name=X --hosts X --auth X
+.PP
+3. Running discovery, such as:
+.PP
+.B rho scan --profile=X --reportfile=Y
+.PP
+The following sections cover these commands in more detail.
+.PP
+The primary purpose of rho is to scan a network. This can be done using simply 'rho scan' and passing all of the required network parameters and authentication credentials with every discovery pass. Using network, authentication, and report profiles, however, simplifies discovery.
```

Comment:
I don't think it's currently possible to pass all of the parameters as command-line arguments.